solcx/wrapper.py

```diff
@@ -46,9 +45,10 @@ def solc_wrapper(
     if solc_binary is None:
         solc_binary = get_executable()
 
-    command = [solc_binary]
+    filename = os.path.split(solc_binary)[-1]
```

Comment:
Prefer to use `pathlib.Path` objects for handling paths - they handle windows paths better and the syntax is cleaner, imo. In this case:

```python
Path(solc_binary).name
```

Reply:
Turns out this was actually breaking with Windows versions anyway, as the filename needs to be the parent directory and not the `.exe` file